README.md

```diff
@@ -854,6 +852,11 @@ Other Style Guides
     function f2(obj) {
       const key = Object.prototype.hasOwnProperty.call(obj, 'key') ? obj.key : 1;
     }
+
+    // best
+    function f2(obj) {
+      const key = Object.hasOwn(obj, 'key') ? obj.key : 1;
+    }
```

Comment:
unfortunately until https://npmjs.com/airbnb-es-shims includes this polyfill, it can't be recommended here.